### ISSUE TITLE
User sign-up, login and posting system

### ISSUE THREAD

**Issue by helimike** [bshaffer/oauth2-server-php#516]
Hi
I am building an app like Instagram and this is my first post at Github. Users can sign up and login, once inside they can post image, write comments and so on. I have followed your nice walkthough and are able to receive an access token based on user credentials. Should I expand the oauth2-server to handle all my database request, like adding users and images to an app_images table or should that be a stand-alone system? I see that there already is a function for adding users.

/Mike

**Comment by thunderyup** [comment]
Use this as a library to your application.  i.e. an addition.  It's not good practice to take it and extend it to *be* your application.

**Comment by bshaffer** [comment]
Yes, @thunderyup is right. This library is not meant to be an application
framework, but yet to work inside any multitude of other frameworks. Use
one of those for your application, and integrate this oath library into it.

On Friday, March 13, 2015, thunderyup <notifications@github.com> wrote:

> Use this as a library to your application. i.e. an addition. It's not good
> practice to take it and extend it to *be* your application.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bshaffer/oauth2-server-php/issues/516#issuecomment-79302290>
> .
>


-- 
Sincerely,
Brent Shaffer
@bshaffer <http://twitter.com/bshaffer>
